README.md

```diff
@@ -50,17 +50,41 @@ this.$sentry.captureException(new Error('example'))
 
 where `this` is a Vue instance.
 
-### Usage Server-Side
+### Usage in the `fetch` and `asyncData` methods
 
-While server-side, `Sentry` is available as `ctx.app.$sentry` where `ctx` is the app context. If you wanted to use it in the `asyncData` method you could do it this way: 
+While using nuxt's `fetch` and `asyncData` methods, the `$sentry` object is available in the root Vue object `app` like other nuxt plugins:
```

Comment:
`app` is an object with root component **options**, not the root Vue object itself. Naming is a bit misleading.

https://nuxtjs.org/api/context